packages/create-llama/questions.ts

```diff
@@ -389,6 +396,11 @@ export const askQuestions = async (
         },
         handlers,
       );
+      // Initialize with default config
+      program.dataSource = {
```

Comment:
@leehuwuj this is sufficient as you already define the default above:
```suggestion
      // Initialize with default config
      program.dataSource = getPrefOrDefault("dataSource");
```